backend/src/main/kotlin/no/uib/echo/schema/Feedback.kt

```diff
@@ -0,0 +1,26 @@
+package no.uib.echo.schema
+
+import kotlinx.serialization.Serializable
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.Table
+
+@Serializable
+data class FeedbackJson(val email: String?, val name: String?, val message: String)
+
+@Serializable
+data class FeedbackResponseJson(val email: String?, val name: String?, val message: String, val sent: String)
+
+enum class FeedbackResponse {
+    EMPTY,
+    SUCCESS,
+}
+
+object Feedback : Table() {
+    private val id: Column<Int> = integer("id").uniqueIndex().autoIncrement()
+    val email: Column<String?> = text("email").nullable()
+    val name: Column<String?> = text("name").nullable()
+    val message: Column<String> = text("message")
+    val sent: Column<String> = text("sent")
```

Comment:
Burde bruke `datetime()` her i stedet for `text()`.
Det letteste er å gjøre som i `Registration`:
```kotlin
val sent: Column<DateTime> = datetime("sent").defaultExpression(CurrentDateTime)
```
Da vil den også automatisk fylle ut tidspunkt uten at man trenger å oppdatere `sent`.